README.md

```diff
@@ -3,3 +3,11 @@
 ###### org.hypertrace.viewgenerator
 
 [![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-view-generator.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-view-generator)
+
+This repo provides two jobs: *1) Hypertrace view creator job 2) Hypertrace view generator job*
+
+Hypertrace view creator:
+It is a bootstrap job that creates require views in pinot like spanEventView, backendEntityView, etc - 
```

Comment:
s/require/required/

Reply:
protip, unless reviewing on phone, where agree this is difficult, go ahead and click this to make feedback like this easy peasy.

![Screen Shot 2020-08-19 at 9 20 35 AM](https://user-images.githubusercontent.com/64215/90581262-6541ad80-e1fd-11ea-9235-739e78d6c9c4.png)

Ends up like this:

```suggestion
It is a bootstrap job that creates required views in pinot like spanEventView, backendEntityView, etc - 
```

Reply:
Ahh.. that's nice. Have never used this feature but it's nice.